cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -199,6 +199,7 @@ func Run(s *options.CMServer) error {
 	panic("unreachable")
 }
 
+// ControllerContext is general controller`s struct
```

Comment:
I agree there should be a comment for this struct, but the comment should explain the purpose of the struct, or describe usage--i.e., add detail one can't infer just from the name of the struct.

(also, a nit: use a regular ' instead of the backtick ` to be more consistent with the rest of the code base)